### ISSUE TITLE
Intel Mac Version

### ISSUE THREAD

**Issue by ejunker** [toocool/bolter-desktop-app#1]
I'm still using an Intel based Mac. It would be nice to have a build that can run on Intel Macs.

**Comment by toocool** [comment]
@ejunker Yes soon it will be built for Intel versions, currently we don't have the Apple Developer program to license and make the apps for Apple official. The Beta version will have the builds for all OS even Linux and Windows
